src/v1/client.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiWrapper } from "./api-wrapper";
```

Comment:
This is common code, we can bring outside V1

Reply:
This is not a common file once API contract change, this file is going to be hot directly.